openpower/configs/linux/vesnin_fragment

```diff
@@ -0,0 +1,2 @@
+CONFIG_QED=m
+CONFIG_QEDE=m
```

Comment:
I suggest putting this in `openpower/configs/linux/skiroot_defconfig`

Reply:
Ok, I will put them into common config.
But we have to decide what we will do with firmware files, I found out that the Linux kernel we use sometimes is newer than linux-firmware package in buildroot (see below).